src/apps/concepts/api.ts

```diff
@@ -35,6 +35,7 @@ const api = {
             sortBy?: string;
             dataTypeFilters?: string[];
             classFilters?: string[];
+            generalFilters?: string[];
             sourceFilters?: string[];
             includeRetired?: boolean;
```

Comment:
This would then be of type number. `includeRetired?: number`

Reply:
@hadijahkyampeire thanks for the review! concerning `Also my understanding is that what you are calling generalFilters, should instead be includeRetired, which becomes true and false on checking and unchecking the checkbox respectively.`  kindly look at the ticket requirement critically its not the way you are suggesting.

Reply:
We definitely don't need a "generalFilters" that in the API (the other filters are actually sent directly to the backend; in this case we just need to change the `includeRetired` toggle which is already defined on line 40).